src/main/prereqs.ts

```
@@ -0,0 +1,90 @@
+import fs from "fs";
+import { https } from "follow-redirects";
+import { logger } from "@/main/logger";
+import { userPreferences, USER_PREFERENCE_KEYS } from "@/main/config";
+import { existsSync } from "fs-extra";
+
+export const netDesktopFileName = "windowsdesktop-runtime-6.0.1-win-x64.exe";
+export const netConsoleFileName = "dotnet-runtime-5.0.13-win-x64.exe";
+export const cLibFileName = "VC-redist.x64.exe";
+
+const installFile = async (url: string, fileName: string) => {
+  https.get(url, (res) => {
+    const path = `${userPreferences.get(
+      USER_PREFERENCE_KEYS.MOD_DIRECTORY
+    )}/launcher/prereqs/${fileName}`;
+    const fileStream = fs.createWriteStream(path);
+    res.pipe(fileStream);
+
+    fileStream.on("error", (err) => {
+      logger.debug("Error: " + err);
```

Should we also throw a JavaScript error here so we can catch it correctly?